src/components/Comments/Comment.js

```diff
@@ -130,14 +132,14 @@ const Comment = ({
   };
 
   const username = comment?.data?.anon_user_info_color_name
-    ? `Anonymous ${comment.data?.anon_user_info_emoji_name}`
+    ? `${comment.data.anon_user_info_color_name} ${comment.data?.anon_user_info_emoji_name}`
     : user?.data?.username;
 
   const onPressDM = async () => {
     try {
       setLoading({...loading, loadingDm: true});
       if (!comment?.data?.anon_user_info_color_name) {
-        const channelName = [username, profile?.myProfile?.username].join(',');
+        const channelName = username;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for generating the `channelName` has been changed. Ensure that this change doesn't break any existing functionality or violate any business rules.



<!-- This is an auto-generated comment by OSS CodeRabbit -->